app/controllers/api/subcollections/request_tasks.rb

```diff
@@ -9,7 +9,7 @@ def request_tasks_query_resource(object)
       def request_tasks_edit_resource(_object, type, id = nil, data = {})
         raise BadRequestError, "Must specify an id for editing a #{type} resource" unless id
         request_task = resource_search(id, type, collection_class(:request_tasks))
-        request_task.update_attributes(:options => request_task.options.merge(data['options'] || {}))
+        request_task.update_attributes(:options => request_task.options.merge(data['options'].deep_symbolize_keys || {}))
```

Comment:
@imtayadeway data['options'] could be missing should the deep symbolize_keys be outside the round brackets?

Reply:
@imtayadeway we use deep_symbolize_keys here but for request we use symbolize_keys?
https://github.com/ManageIQ/manageiq-api/blob/f257e71ea7c78063659efe2d5bc77c8cc14f659f/lib/services/api/request_parser.rb#L13

Reply:
@mkanoor I updated it so it's a bit easier to read/follow